wiki/Help_centre/Website/de.md

```diff
@@ -0,0 +1,106 @@
+---
+tags:
+  - site
+  - block
+  - forum
+  - ticket
+  - appearance
+  - message
+  - page
+  - profile
+  - missing
+  - stats
+  - Webseite
+  - Aussehen
+  - Nachricht
+  - Profil
+  - fehlt
+---
+
+# Webseite
+
+*Hauptseite: [Hilfecenter](/wiki/Help_centre)*
+
+Diese Sektion befasst sich mit Problemen auf der osu!-Webseite.
+
+## Häufige Probleme
+
+### Ich habe keine Antwort auf mein Support-Ticket erhalten und es sind schon über 2 Wochen vergangen!
+
+**In der überwiegenden Mehrheit der Fälle liegt das an einer hohen Zahl an Tickets, die zur gleichen Zeit in unser System eingehen.**
+
+Du kannst immer eine einzelne Antwort auf dein aktuell offenes Ticket senden, um den Status deines Tickets anzufragen, aber stelle sicher, dass du dies nicht mehr als zwei- oder dreimal pro Woche machst. Ebenfalls kannst du uns auf Twitter unter [@osusupport](https://twitter.com/osusupport) kontaktieren, um nach deinem Ticket zu fragen.
+
+In seltenen Umständen können Support-Tickets von <!--a-->chronisch straffälligen Nutzern absichtlich ignoriert werden, jedoch wird sich das [Support-Team](/wiki/People/The_Team/Account_support_team) vorher **immer** bei dir melden, wenn sie nicht mehr länger antworten werden. Falls du noch keine Benachrichtigung wie diese erhalten hast, brauchst du dir keine Sorgen machen!
```

Comment:
```suggestion
In seltenen Umständen können Support-Tickets von chronisch straffälligen Nutzern absichtlich ignoriert werden, jedoch wird sich das [Support-Team](/wiki/People/The_Team/Account_support_team) vorher **immer** bei dir melden, wenn sie nicht mehr länger antworten werden. Falls du noch keine Benachrichtigung wie diese erhalten hast, brauchst du dir keine Sorgen machen!
```
Wanted to check on `chronisch straffällig` once again, as I've always heard `chronisch` in the context of illnesses and rarely/never in such a case. I can't find a better translation for this though and deepl suggests the same.

Reply:
There is the term `chronic offender` which could be translated to `Gewohnheitsverbrecher`, but I think it's also fine as is. 
```suggestion
In seltenen Umständen können Support-Tickets von Gewohnheitsverbrechern absichtlich ignoriert werden, jedoch wird sich das [Support-Team](/wiki/People/The_Team/Account_support_team) vorher **immer** bei dir melden, wenn sie nicht mehr länger antworten werden. Falls du noch keine Benachrichtigung wie diese erhalten hast, brauchst du dir keine Sorgen machen!
```

Reply:
Easiest translation would be `wiederholt straffällig` I think. Or maybe offending is meant to mean `beleidigend`/insulting in this context? As in you don't get ignored for breaking the rules of the game, you get ignored for being rude to the support team...

Reply:
I think so too. The phrase refers to people that are rude and insulting to the support team several times. I will leave the translation as-is 👍